src/views/AboutMonarch.md

```diff
@@ -1,137 +1,82 @@
-<div
-  class="container-fluid monarch-view monarch-about-view">
+<div class="container-fluid monarch-view monarch-about-view">
 
-#### Goals
```

Comment:
The GOALS part should be preserved. For now let's leave this fragment as is (lines 4 through 18 in this file), after the introduction (to appear after "translational research" at the end of the section you are inserting). We'll have to get more feedback from @jmcmurry later on, in case she wants to modify this text.